<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSigGen</class>
 <widget class="QWidget" name="FormSigGen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>262</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>245</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>262</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Signal Generator</string>
  </property>
  <widget class="QPushButton" name="pushButtonStartStop">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCh2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel 2</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxAmp1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>-60.000000000000000</double>
   </property>
   <property name="maximum">
    <double>0.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>-5.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelPhi2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Phi2 [deg]:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAmp2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Amplitude2 [dBFS ]:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxFunc">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelAmp1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Amplitude [dBFS ]:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFreq1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequency [Hz]:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCh1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel 1</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Channel 1</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBoxFreq2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>20000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
   <property name="value">
    <number>7000</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelFunc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Function:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPhi1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Phi [deg]:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxFreq1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>20000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>75</number>
   </property>
  </widget>
  <widget class="QWidget" name="widgetGraph" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>240</width>
     <height>200</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPhi1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxAmp2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>-60.000000000000000</double>
   </property>
   <property name="maximum">
    <double>0.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>-17.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelFreq2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequency2 [Hz]:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPhi2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Output Channel:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBoxFunc</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>spinBoxFreq1</tabstop>
  <tabstop>doubleSpinBoxAmp1</tabstop>
  <tabstop>spinBoxPhi1</tabstop>
  <tabstop>spinBoxFreq2</tabstop>
  <tabstop>doubleSpinBoxAmp2</tabstop>
  <tabstop>spinBoxPhi2</tabstop>
  <tabstop>pushButtonStartStop</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
